integration-tests/tests/network/runner.rs

```diff
@@ -374,7 +382,7 @@ impl StateMachine {
                                 (nonce, info.read().unwrap().peers_info[source].id.clone(), count)
                             })
                             .collect();
-
+let action_clone2 = action_clone.clone();
```

Comment:
is it related to this PR?

Reply:
This was used to spawn another future. This is not needed anymore.